nvflare/private/fed/server/server_engine.py

```diff
@@ -784,4 +784,5 @@ def server_shutdown(server, touch_file):
         time.sleep(3.0)
     finally:
         server.status = ServerStatus.SHUTDOWN
+        security_close()
```

Comment:
should this `security_close` be called in the same place as `security_init`?

In server_train.py ?